app/src/main/kotlin/cz/covid19cz/erouska/ui/update/efgs/EfgsUpdateFragment.kt

```diff
@@ -63,6 +63,9 @@ class EfgsUpdateFragment : BaseFragment<FragmentEfgsUpdateBinding, EfgsUpdateVM>
             }
         }
 
+        // turn on EFGS automatically on this screen
+        viewModel.sharedPrefsRepository.setTraveller(true)
```

Comment:
Tohle mi přijde hodně divné, většina lidí necestuje a tak by neměla být traveller. Chce to víc diskuze s Honzou Štíchou.

Reply:
Souhlasím, vykomunikujte prosím s Honzou / Vlastou. Jsem oof tento víkend.

Reply:
Diskuze ještě potrvá a potřebujeme ty jiné věci, tak odebírám z této branche. Příště prosím pro každou JIRA issue jinou branch.